packages/Teams/js/package.json

```diff
@@ -29,27 +29,27 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "peerDependencies": {
-    "adaptive-expressions": "~4.14.0-rc0",
-    "botbuilder": "~4.14.0-rc0",
-    "botbuilder-dialogs": "~4.14.0-rc0",
-    "botbuilder-dialogs-adaptive": "~4.14.0-preview.rc0",
-    "botbuilder-dialogs-adaptive-runtime-core": "~4.14.0-preview.rc0",
-    "botbuilder-dialogs-declarative": "~4.14.0-preview.rc0",
-    "botbuilder-stdlib": "~4.14.0-internal.rc0",
+    "adaptive-expressions": "~4.14.0-rc1",
+    "botbuilder": "~4.14.0-rc1",
+    "botbuilder-dialogs": "~4.14.0-rc1",
+    "botbuilder-dialogs-adaptive": "~4.14.0-preview.rc1",
+    "botbuilder-dialogs-adaptive-runtime-core": "~4.14.0-preview.rc1",
+    "botbuilder-dialogs-declarative": "~4.14.0-preview.rc1",
+    "botbuilder-stdlib": "~4.14.0-internal.rc1",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
     "@tsconfig/recommended": "^1.0.1",
     "@types/lodash": "^4.14.168",
     "@typescript-eslint/eslint-plugin": "latest",
     "@typescript-eslint/parser": "latest",
-    "adaptive-expressions": "4.14.0-rc0",
-    "botbuilder": "4.14.0-rc0",
-    "botbuilder-dialogs": "4.14.0-rc0",
-    "botbuilder-dialogs-adaptive": "~4.14.0-preview.rc0",
-    "botbuilder-dialogs-adaptive-runtime-core": "~4.14.0-preview.rc0",
-    "botbuilder-dialogs-declarative": "~4.14.0-preview.rc0",
-    "botbuilder-stdlib": "~4.14.0-internal.rc0",
+    "adaptive-expressions": "~4.14.0-rc1",
```

Comment:
Technically (not a major issue since these are dev deps only) these should all lock to a specific version without a semver operator. We rely on `instanceof` in a few places which can break when transitive versions differ slightly.

Reply:
To clarify, lock all of the bot-related `devDependencies`? How about `peerDependencies`?

Reply:
Just dev deps. We want to allow users to use any compatible versions, and we assume they can ensure the compatible version they select is consistent across all SDK packages.